src/core/umount.c

```diff
@@ -46,6 +49,9 @@
 #include "util.h"
 #include "virt.h"
 
+DEFINE_TRIVIAL_CLEANUP_FUNC(struct libmnt_table*, mnt_free_table);
+DEFINE_TRIVIAL_CLEANUP_FUNC(struct libmnt_iter*, mnt_free_iter);
```

Comment:
maybe add libmount-util.h for this?

Reply:
Let's wait until the situation with libmount.h is cleared.